src/app/shared/issue-tables/issue-tables.component.html

```diff
@@ -145,6 +149,27 @@
     </mat-cell>
   </ng-container>
 
+  <!--Tester Disagree Column-->
+  <ng-container matColumnDef="testerDisagree">
+    <mat-header-cell mat-header-cell *matHeaderCellDef mat-sort-header>Response</mat-header-cell>
+    <mat-cell *matCellDef="let issue">
+      <span
+        (click)="$event.stopPropagation()"
+        *ngIf="issue.testerDisagree"
+        [ngStyle]="this.labelService.setLabelStyle(this.labelService.getColorOfLabel('Rejected'))"
```

Comment:
Since you're going to use the Accepted/Rejected label colours, maybe can shift the refactoring of the colour constants to another PR? Bearing in mind prof's comment on my PR on only changing what is necessary in a PR.

Sorry for making you refactor the colour constants :(

Reply:
Np, will undo these changes on colours and create another PR

Using Accepted Rejected label colours help ensure a singular reference to colour hexes after all